src/components/orgSelector.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import { useAllOrgs } from "dcs-react-hooks";
+import Autocomplete from "@mui/material/Autocomplete";
+import TextField from "@mui/material/TextField";
+
+export default function Organization() {
+
+  const { state, actions } = useAllOrgs({ lang: "en", page: 1, limit: 5 });
+  const { orgs, isLoading } = state;
+  const [orgName, setOrgName] = useState("");
```

Comment:
You can remove this internal state and use the passed in props of `organization` and `onOrganization` instead.